cc/tf_bt_utils.cc

```diff
@@ -141,6 +141,8 @@ void WriteEvalRecord(const std::string& gcp_project_name,
       SetCell("metadata", "length", absl::StrCat(game.moves().size())),
       SetCell("metadata", "sgf", sgf_name), SetCell("metadata", "tag", tag));
 
+  // NOTE: In addition to metadata also eval_games column family.
```

Comment:
Incomplete sentence?

Reply:
removed. I think I'll remember and the whole design is flexible.